tests/host/Makefile

```diff
@@ -51,8 +51,8 @@ TEST_CPP_FILES := \
 	core/test_md5builder.cpp \
 
 
-CXXFLAGS += -std=c++11 -Wall -coverage -O0 -fno-common
-CFLAGS += -std=c99 -Wall -coverage -O0 -fno-common
+CXXFLAGS += -std=c++11 -Wall -coverage -O0 -fno-common -g
```

Comment:
Is this -g intentional, or left over from debugging?

Reply:
I can pull it out.  At some point I want to add in valgrind as part of all host_tests, which will require it back in.  We're building w/`-o0` already so it's kind of a strange setup already...